build

```diff
@@ -204,6 +268,11 @@ function parse_config () {
 			if [ "$?" -ne 0 ]; then
 				exit 1
 			fi
+		elif [[ "$section" == "github" ]]; then
+			parse_github_repo $1
+			if [ "$?" -ne 0 ]; then
+				exit 1
+			fi
 		elif [[ "$section" == "proxy" ]]; then
```

Comment:
Hi Ngoan,
this is deprecated...
Thank you,
 Thomas

Reply:
Hi Thomas.

I have removed this deprecated config in this commit https://github.com/test-fullautomation/RobotFramework_AIO/pull/16/commits/103355d6a13950d9decae09cdd56623732e588d5

Thank you,
Ngoan